tests/subsys/fs/fs_api/src/test_fs_dir_file.c

```diff
@@ -142,16 +272,24 @@ void test_file_statvfs(void)
 
 	ret = fs_statvfs(NULL, &stat);
 	zassert_not_equal(ret, 0, "Get valume without path");
+	ret = fs_statvfs(TEST_FS_MNTP, NULL);
+	zassert_not_equal(ret, 0, "Get valume without stat structure");
+
+	ret = fs_statvfs("/", &stat);
+	zassert_not_equal(ret, 0, "Get valume by too short path");
+
+	ret = fs_statvfs("SDCARD:", &stat);
+	zassert_not_equal(ret, 0, "Get valume by path without /");
```

Comment:
In neither of these you are attempting to "Get valume [sic]", you are attempting to get volume statistics, by the function called.